WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerAutomatticTracks.m

```diff
@@ -1490,7 +1490,7 @@ + (TracksEventPair *)eventPairForStat:(WPAnalyticsStat)stat
             eventName = @"reader_site_followed";
             break;
         case WPAnalyticsStatReaderSitePreviewed:
-            eventName = @"reader_blog_preview";
+            eventName = @"reader_blog_previewed";
```

Comment:
Important info: we almost never rename events. The reason is that there's no "merge" in the Analytics tool, so if we do it we end up with two different events and that can possibly affect a lot of calculations, funnels, etc.

In Android the name of the event is `reader_blog_previewed` but AFAIK it has always been like that. It's a shame that both events don't have the same name but we should keep `reader_blog_preview`.

Reply:
Ughhhh, you're right. In that case, the ship has sailed, I agree with you, let's leave as it is.